docs/gaming/game-examples.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Learn By Example
+head:
+  - - meta
+    - name: title
+      content: Solana Cookbook | Learn By Example
+  - - meta
+    - name: og:title
+      content: Solana Cookbook | Learn By Example
+  - - meta
+    - name: description
+      content: A list of open source games with tutorials to get you started.
+  - - meta
+    - name: og:description
+      content: A list of open source games with tutorials to get you started.
+  - - meta
+    - name: og:image
+      content: https://solanacookbook.com/cookbook-sharing-card.png
+  - - meta
+    - name: og:image:alt
+      content: Solana splash card
+  - - meta
+    - name: twitter:card
+      content: summary
+  - - meta
+    - name: twitter:site
+      content: "@solanacookbook"
+  - - meta
+    - name: twitter:image
+      content: "https://solanacookbook.com/cookbook-sharing-card.png"
+  - - meta
+    - name: robots
+      content: index,follow,noodp
+  - - meta
+    - name: googlebot
+      content: index,follow
+footer: MIT Licensed
+---
+
+## An unordered list of open source games as reference for learning
+
+## Interact with Anchor Program from Unity
+
+A simple example moving a player left and right using Anchor framework and Unity SDK
+<br/>
```

Comment:
Why all the `<br/>`s? Can't you get this with normal line breaks?

Reply:
<img width="275" alt="image" src="https://user-images.githubusercontent.com/5938789/235456070-ba3bc119-3be3-4f25-a86b-4188581d0740.png">
Without the extra linebreaks it doesnt look very nice.

Reply:
If you just put two line breaks it would work without the `<br/>`

Example:
```
[Example Source Code](https://github.com/Woody4618/SolPlay_Unity_SDK/blob/main/Assets/SolPlay/Scripts/Services/WalletHolderService.cs)


[Example Game](https://solplay.de/SolHunter/index.html)


```

Reply:
Yeah thats the second picture i posted. It's just, that it's wasting some space. But if you like that better i'll change it to that.